examples/README.md

```diff
@@ -0,0 +1,19 @@
+# Versatile Data Kit Examples
```

Comment:
have you updated your local git repo? You added this file to the previous scenario already.

Reply:
Sorry...I'm going to update it...